packages/starknet-snap/src/utils/rpc.test.ts

```diff
@@ -30,6 +30,15 @@ const validateParam = {
     '0x04882a372da3dfe1c53170ad75893832469bf87b62b13e84662565c4a88f25cd',
 };
 
+const treeStruct = union([
```

Comment:
it is better to use superstruct refine, rather than on RPC or
use the union from metamask/utils